cpp/src/Slice/Parser.cpp

```diff
@@ -694,36 +694,34 @@ namespace
         if (l.find(tag) == 0)
         {
             const string ws = " \t";
+            string::size_type pos;
 
             if (namedTag)
             {
-                string::size_type n = l.find_first_not_of(ws, tag.size());
-                if (n == string::npos)
+                pos = l.find_first_not_of(ws, tag.size());
+                if (pos == string::npos)
                 {
                     return false; // Malformed line, ignore it.
                 }
-                string::size_type end = l.find_first_of(ws, n);
+                string::size_type end = l.find_first_of(ws, pos);
                 if (end == string::npos)
                 {
                     return false; // Malformed line, ignore it.
                 }
-                name = l.substr(n, end - n);
-                n = l.find_first_not_of(ws, end);
-                if (n != string::npos)
-                {
-                    doc = l.substr(n);
-                }
+                name = l.substr(pos, end - pos);
+                pos = end;
             }
             else
             {
                 name.clear();
+                pos = tag.size();
+            }
 
-                string::size_type n = l.find_first_not_of(ws, tag.size());
-                if (n == string::npos)
-                {
-                    return false; // Malformed line, ignore it.
```

Comment:
Returning `false` if the line was empty wasn't good behavior.

It would cause the comment parser to think this was unknown tag, not a malformed tag.
And seeing `unknown doc comment tag '@see'` is very confusing.

There wasn't even a point to returning false, since all the calling code gracefully handles empty lines anyways.